server/rpc/server_test.go

```diff
@@ -22,29 +22,39 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"reflect"
 	"testing"
 	"time"
 
 	"github.com/gogo/protobuf/types"
 	"github.com/stretchr/testify/assert"
+	monkey "github.com/undefinedlabs/go-mpatch"
```

Comment:
When running this test on Apple Silicon machine, the following error occurs.

How about separating admin tests with another file with `//go:build amd64` tag?

```
--- FAIL: TestAdminRPCServerBackend (10.07s)
    --- FAIL: TestAdminRPCServerBackend/admin_remove_document_consistence_test (0.74s)
        server_test.go:1265:
            	Error Trace:	/Users/hackerwins/Development/yorkie-team/yorkie/server/rpc/server_test.go:1265
            	Error:      	Received unexpected error:
            	            	unsupported architecture: arm64
            	Test:       	TestAdminRPCServerBackend/admin_remove_document_consistence_test
        server_test.go:1297:
            	Error Trace:	/Users/hackerwins/Development/yorkie-team/yorkie/server/rpc/server_test.go:1297
            	Error:      	Received unexpected error:
            	            	unsupported architecture: arm64
            	Test:       	TestAdminRPCServerBackend/admin_remove_document_consistence_test
        server_test.go:1340:
            	Error Trace:	/Users/hackerwins/Development/yorkie-team/yorkie/server/rpc/server_test.go:1340
            	Error:      	Received unexpected error:
            	            	rpc error: code = Unimplemented desc = method GetProject not implemented
            	Test:       	TestAdminRPCServerBackend/admin_remove_document_consistence_test
        server_test.go:1351:
            	Error Trace:	/Users/hackerwins/Development/yorkie-team/yorkie/server/rpc/server_test.go:1351
            	Error:      	Received unexpected error:
            	            	rpc error: code = FailedPrecondition desc = client(648183dc4afe301f8a574115) attaches document(648183dc4afe301f8a57411f): document already attached
            	Test:       	TestAdminRPCServerBackend/admin_remove_document_consistence_test
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
```